doc/files.md

```diff
@@ -57,6 +57,7 @@ Subdirectory       | File(s)               | Description
 `./`               | `guisettings.ini.bak` | Backup of former [GUI settings](#gui-settings) after `-resetguisettings` option is used
 `./`               | `mempool.dat`         | Dump of the mempool's transactions
 `./`               | `onion_private_key`   | Cached Tor onion service private key for `-listenonion` option
+`./`               | `onion_private_key_obsolete_torv2` | Obsolete Tor onion service private key used to create TORv2 hidden services. This file is automatically created as a backup when new TORv3 private key is generated and saved into `onion_private_key`. Bitcoin Core does not need it and will never attempt to read it. Remove if you do not need the old TORv2 hidden service.
```

Comment:
I'd slightly prefer creating a new file for v3 instead of moving the old one. This makes it easier to go back and forth to test this.

Reply:
Indeed. Done!